### PR TITLE
Code update for inference

### PR DESCRIPTION
**Pull request by jeewonkimm2**
I created a file called '[test_single_edit.py](https://github.com/jeewonkimm2/generative-inpainting-pytorch/blob/jeewon/test_single_edit.py)'.

- Overview : Enable inpainting at the coordinates you have specified.
1. Download checkpoint from (Trained PyTorch model: [[Google Drive](https://drive.google.com/open?id=1qbfA5BP9yzdTFFmiOTvYARUYgW1zwBBK)] [[Baidu Wangpan](https://pan.baidu.com/s/17HzpiqMPLIznvCWBfpNVGw)]) and save the entire folder called 'checkpoints'.
2. Input image size should be [256, 256].
3. Specify the coordinates(x1, y1, x2, y2) of inpainting part when you run the file.

```bash
python test_single_edit.py \
	--image examples/imagenet/imagenet_patches_ILSVRC2012_val_00008210_input.png \
	--output examples/output.png \
	--x1 100 \
	--y1 50 \
	--x2 180 \
	--y2 210
```
- --image : Path of an input image
- --output : Path of result
- x1, y1, x2, y2 : Coordinates

### CODE REVIEW

File: .DS_Store
Comment (doorooful) [inline .DS_Store:1]: Can you add this file format in gitignore?

File: data/.DS_Store
Comment (doorooful) [inline data/.DS_Store:1]: Same here!

File: model/.DS_Store
Comment (doorooful) [inline model/.DS_Store:1]: Same here!

File: utils/.DS_Store
Comment (doorooful) [inline utils/.DS_Store:1]: Same here!

**Review by doorooful — COMMENTED** [review]
Please update `.gitignore` file regarding mac.

**Comment by jeewonkimm2** [comment]
I inserted them in `.gitignore`

```bash
.DS_Store
data/.DS_Store
model/.DS_Store
utils/.DS_Store
```

Can you check it please?

**Review by doorooful — APPROVED** [review]
Thank you!

**Pull request merged**